app/components/task_list_item_references_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul class="govuk-list">
+  <% references.each do |reference| %>
+    <li class="govuk-body-s">
+      <span class="app-status-indicator app-status-indicator--<%= colour_for(reference) %>">
+      </span>&nbsp;
+      <b><%= reference.name %></b>: <%= status_label_for(reference) %>
```

Comment:
Can this be on one line? Affects the space between the indicator and the name, otherwise.

```suggestion
      <span class="app-status-indicator app-status-indicator--<%= colour_for(reference) %>"></span>&nbsp;<b><%= reference.name %></b>: <%= status_label_for(reference) %>
```